lib/rspec/rails/view_rendering.rb

```diff
@@ -81,7 +81,11 @@ def append_view_path(new_path)
       private
 
         def _path_decorator(path)
-          EmptyTemplateResolver.new(path)
+          if path.is_a?(::ActionView::Resolver)
+            path
```

Comment:
Why? If the point here is nullify view rendering surely it's better to continue to ignore this?